package.json

```diff
@@ -19,6 +19,7 @@
     "@emotion/styled": "^11.1.5",
```

Comment:
looks like the package-lock changes are still huge - i cant event view them in the browser. 

do you know why that is? are you using a different version of npm? or did you forget to run npm install after the package changes?

Reply:
I am not sure what you mean here. When I ran the npm install command, it installed the latest versions of dependencies and it automatically regenerated lock file. Changes in lock file are for every dep, because all versions were fixiated before and now are on the latest version.

Reply:
are you using npm 6?

try reverting your changes to package-lock and running npm install with npm 6. the only changes to the lock file should be for gatsby-plugin-mailchimp

at the moment, the whole file has  changed and I cant even see the diff

![image](https://user-images.githubusercontent.com/3308076/205325366-ae6cffa1-b637-413f-8e30-9960f39fbed2.png)

Reply:
@odeniso in the readme there were not any instructions on which npm to use. So I used node 16 with npm 8. 

The number of changes is normal here because I first fixated versions and then committed those changes. Then I reverted that change and install it again using npm 8 and committed it. And now with npm 6.

Reply:
cool. now i can see the diff. its a lot of dependency changes for a small change, but no big deal